src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs

```diff
@@ -108,17 +116,24 @@
                    {:text "Running" :pred #(= "Running" (:status %))}
                    {:text "Exception" :pred #(= "Exception" (:status %))}]
          :columns
-         [{:header [:div {:style {:marginLeft -6}} "Status"] :starting-width 60
-           :content-renderer (fn [data] [StatusCell {:data data}])
-           :filter-by :none}
-          {:header "Workspace" :starting-width 450
-           :content-renderer (fn [data] [WorkspaceCell {:data data}])
-           :filter-by :name}
+         [{:sort-by (fn [m] (:status m))
+           :header [:div {:style {:marginLeft -6}} "Status"] :starting-width 60
+           :content-renderer (fn [data] [StatusCell {:data data}]) :filter-by :none}
+          {:sort-by (fn [m] (:name m))
```

Comment:
`:sort-by :name`.

You can also add `:as-text :name`, then use `:sort-by :text` and get rid of the `:filter-by`.